HzMemoryCache/Diagnostics/HzCacheDiagnostics.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace HzCache.Diagnostics
+{
+    public static class HzCacheDiagnostics
+    {
+        public const string HzCacheVersion = "0.0.9";
```

Comment:
Surely this will be 0.0.10 once this is merged?

Reply:
maby we should remove this as no one will keep it updated?